packages/cursorless-engine/src/spokenForms/CustomSpokenForms.ts

```diff
@@ -175,7 +168,7 @@ function updateEntriesForType<T extends SpokenFormType>(
             defaultSpokenForms,
             spokenForms: customSpokenForms,
             requiresTalonUpdate: false,
-            isCustom: isEqual(defaultSpokenForms, customSpokenForms),
+            isCustom: !isEqual(defaultSpokenForms, customSpokenForms),
```

Comment:
whoops 😅. we don't have any use case for this yet, but should prob add a test at some point

Reply:
Really proves that if something isn't tested it can easily be correct :)